src/point_one/fusion_engine/messages/gnss_corrections.h

```diff
@@ -0,0 +1,69 @@
+/**************************************************************************/ /**
+ * @brief GNSS corrections messages.
+ * @file
+ ******************************************************************************/
+
+#pragma once
+
+#include "point_one/fusion_engine/messages/defs.h"
+
+namespace point_one {
+namespace fusion_engine {
+namespace messages {
+
+// Enforce 4-byte alignment and packing of all data structures and values.
+// Floating point values are aligned on platforms that require it. This is done
+// with a combination of setting struct attributes, and manual alignment
+// within the definitions. See the "Message Packing" section of the README.
+#pragma pack(push, 1)
+
+/**
+ * @defgroup gnss_corrections GNSS Corrections Message Definitions
+ * @brief Output messages containing GNSS corrections.
```

Comment:
Suggest "~Output~ Messages containing information about GNSS corrections." since in the future this might have corrections inputs, etc., and also status messages and such. Aside from encoded L-band bytes, I don't know that we will have any output messages actually containing consumable GNSS corrections.